features/command_line/randomization.feature

```diff
@@ -1,37 +1,28 @@
 Feature: Randomization can be reproduced across test runs
 
-  In Ruby, randomness is seeded by calling `srand` and passing it the seed that
-  you want to use. By doing this, subsequent calls to `rand`, `shuffle`,
-  `sample`, etc. will all be randomized the same way given the same seed is
-  passed to `srand`.
+  In Ruby, you can call `srand` for randomness and pass it to the seed you want to use. All subsequent calls to `rand`, `shuffle`,
```

Comment:
> call `srand` for randomness

https://ruby-doc.org/core-2.7.1/Random.html#method-c-srand

> Seeds the system pseudo-random number generator


> `srand` for randomness and pass it to the seed you want to use

This is confusing. Pass what to the seed? How can you pass anything to a number which a seed is?


I believe the wording before this change is more correct.